Jellyfin.Api/Controllers/GenresController.cs

```diff
@@ -63,12 +63,15 @@ public class GenresController : BaseJellyfinApiController
         /// <param name="nameStartsWithOrGreater">Optional filter by items whose name is sorted equally or greater than a given input string.</param>
         /// <param name="nameStartsWith">Optional filter by items whose name is sorted equally than a given input string.</param>
         /// <param name="nameLessThan">Optional filter by items whose name is equally or lesser than a given input string.</param>
+        /// <param name="sortBy">Optional. Specify one or more sort orders, comma delimited.</param>
+        /// <param name="sortOrder">Sort Order - Ascending,Descending.</param>
         /// <param name="enableImages">Optional, include image information in output.</param>
         /// <param name="enableTotalRecordCount">Optional. Include total record count.</param>
         /// <response code="200">Genres returned.</response>
         /// <returns>An <see cref="OkResult"/> containing the queryresult of genres.</returns>
         [HttpGet]
         [ProducesResponseType(StatusCodes.Status200OK)]
+        [AllowAnonymous]
```

Comment:
Doesn't this allow unauthorized access? 🤔

Reply:
That’s what I get for leaving testing code in there